src/protocols/pancake/PancakeSwapService.ts

```diff
@@ -70,7 +71,8 @@ export class PancakeSwapService extends SwapService {
     const depositTokenValue = await getTokenPriceData(poolInfo.lpToken);
     const poolValue = poolBalance * depositTokenValue.usd;
     const emissionScalar = poolInfo.allocPoint.toNumber() / totalAllocPoint.toNumber();
-    const cakeEmission = parseFloat(formatEther(cakePerBlock)) * emissionScalar * BSC_BLOCKS_PER_YEAR * tokenPrice.usd;
+    const cakeEmission =
+      parseFloat(formatEther(cakePerBlock)) * emissionScalar * BinanceSmartChain.BLOCKS_PER_YEAR * tokenPrice.usd;
```

Comment:
looking at this we can probably have BLOCK_PER_YEAR be private on those classes and just use `chain.blockPerYear` here

Reply:
Yeah, that sounds good 👍